package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terraware/web-components",
-  "version": "1.0.52",
+  "version": "1.0.53",
```

Comment:
I missed documenting this (will do that shortly) - storybook changes are not published to npm, version bump here won't capture any new changes in the distribution.